api/overdrive.py

```diff
@@ -819,23 +819,32 @@ def circulation_lookup(self, book):
 
     def update_formats(self, licensepool):
         """Update the format information for a single book.
+
+        Incidentally updates the metadata, just in case Overdrive has
+        changed it.
         """
         info = self.metadata_lookup(licensepool.identifier)
 
         metadata = OverdriveRepresentationExtractor.book_info_to_metadata(
-            info, include_bibliographic=False, include_formats=True)
+            info, include_bibliographic=True, include_formats=True)
```

Comment:
Without this, we'll retrieve the bibliographic information from Overdrive, but we'll throw it away. This buys us a little time, but we're inside a script, not handling an HTTP request, and data quality is more important than time.